docs/reference/query-dsl/rank-feature-query.asciidoc

```diff
@@ -120,31 +159,83 @@ GET test/_search
     }
   }
 }
---------------------------------------------------
+----
 // CONSOLE
 
-[float]
-=== Supported functions
 
-The `rank_feature` query supports 3 functions in order to boost scores using the
-values of rank features. If you do not know where to start, we recommend that you
-start with the `saturation` function, which is the default when no function is
-provided.
+[[rank-feature-top-level-params]]
+==== Top-level parameters for `rank_feature`
+
+`field`::
+(Required, string) <<rank-feature,`rank_feature`>> or
+<<rank-features,`rank_features`>> field used to boost
+<<query-filter-context,relevance scores>>.
+
+`boost`::
++
+--
+(Optional, float) Floating point number used to decrease or increase
```

Comment:
One other note: I noticed that [other query documentation](https://www.elastic.co/guide/en/elasticsearch/reference/7.2/query-dsl-dis-max-query.html) places the information in parentheses after the description.

Reply:
I have an open PR to update that other query documentation: #44703

The format is based on the [API Reference template](https://github.com/elastic/docs/blob/master/shared/api-ref-ex.asciidoc), which changed.

Reply:
Got it, thanks for the context.